packages/dso-toolkit/theme/views/partials/browser/panel-info.nunj

```diff
@@ -35,8 +35,8 @@
             <strong class="Meta-key">Preview:</strong>
             <span class="Meta-value">
                 <ul>
-                    <li><a data-no-pjax href="{{ path(frctl.theme.urlFromRoute('preview', { handle: entity.handle })) }}"><span>With layout</span> {% include "icons/open-in-browser.svg" %}</a></li>
-                    <li><a data-no-pjax href="{{ path(frctl.theme.urlFromRoute('render', { handle: entity.handle })) }}"><span>Component only</span> {% include "icons/open-in-browser.svg" %}</a></li>
+                    <li><a data-no-pjax href="{{ path(frctl.theme.urlFromRoute('preview', { handle: entity.handle })) }}"><span>With layout</span> {% include "icons/new-window.svg" %}</a></li>
```

Comment:
Waar is dit te zien?

Reply:
Geen idee, maar omdat hetzelfde icoon gebruikt wordt, heb ik het hier ook aangepast. Wllicht kan @tfrijsewijk duidelijkheid geven?

Reply:
Nee maakt niet uit, het is zeker goed, alleen vroeg ik me af waar dit terechtkwam.